modules/docs/arrow-docs/docs/_data/sidebar.yml

```diff
@@ -35,18 +35,11 @@ options:
       - title: Type Classes
         url: /docs/apidocs/arrow-typeclasses/
 
+      # TODO: Add title and Url for arrow-core, arrow-ui, arrow-mtl-data
```

Comment:
This needs to be completed for the PR to be considered done.

Reply:
I open for discussion

Reply:
We moved this to an issue: 
I will create it and write down steps, maybe other contributors can help me and We can review it step by step